module.combodo-workflow-graphical-view.php

```diff
@@ -38,6 +38,7 @@
 		'datamodel' => array(
 			// Module's autoloader
 			'vendor/autoload.php',
+			'src/Portal/Controller/LifecycleBrickController.php',
```

Comment:
Why did you add this line? It should not be necessary as there is an autoloader. I think you just forgot to re-dump the autoloaders.

Reply:
I wanted to put this file in autoload but I had the error message :

Reply:
We rather fix this issue than force the file to be always loaded.